packages/jellyfish-api-core/src/category/mining.ts

```diff
@@ -100,3 +120,33 @@ export interface SmartFeeEstimation {
   errors?: string[]
   blocks: number
 }
+
+export interface TemplateRequest {
+  mode?: string
+  capabilities?: string[]
+  rules: string[]
+}
+
+export interface BlockTemplate {
+  capabilities: string[]
+  version: number
+  rules: string[]
+  vbavailable: JSON
+  vbrequired: number
+  previousblockhash: string
+  transactions: Transaction[]
+  coinbaseaux: JSON
```

Comment:
```suggestion
  coinbaseaux: any
```

`JSON` isn't really a type.